dex/testing/loadbot/go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/dchest/blake2b v1.0.0 // indirect
 	github.com/dchest/siphash v1.2.3 // indirect
 	github.com/dcrlabs/neutrino-bch v0.0.0-20220809174944-921b271ba678 // indirect
+	github.com/dcrlabs/neutrino-ltc v0.0.0-20220819181220-04c154bb8ed8 // indirect
```

Comment:
Just noticing that loadbot's go.mod still has the `replace github.com/gcash/neutrino => github.com/buck54321/neutrino-bch`